tests/network/primary_pod_network.go

```diff
@@ -70,25 +70,69 @@ var _ = SIGDescribe("[Serial]Primary Pod Network", func() {
 		})
 
 		Context("VMI connected to the pod network using bridge binding", func() {
-			var vmi *v1.VirtualMachineInstance
+			When("Guest Agent exists", func() {
+				var (
+					vmi   *v1.VirtualMachineInstance
+					vmiIP = func() (string, error) {
+						vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+						if err != nil {
+							return "", err
+						}
+						return vmi.Status.Interfaces[0].IP, nil
+					}
 
-			BeforeEach(func() {
-				vmi = setupVMI(virtClient, vmiWithBridgeBinding())
-			})
+					vmiIPs = func() ([]string, error) {
+						vmi, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+						if err != nil {
+							return nil, err
+						}
+						return vmi.Status.Interfaces[0].IPs, nil
+					}
+				)
+				BeforeEach(func() {
+					var err error
 
-			AfterEach(func() {
-				cleanupVMI(virtClient, vmi)
+					vmi, err = newFedoraWithGuestAgentAndDefaultInterface(libvmi.InterfaceDeviceWithBridgeBinding())
+					Expect(err).NotTo(HaveOccurred())
+
+					vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+					Expect(err).NotTo(HaveOccurred())
+					tests.WaitForSuccessfulVMIStart(vmi)
+					tests.WaitAgentConnected(virtClient, vmi)
+				})
+
+				AfterEach(func() {
+					cleanupVMI(virtClient, vmi)
+				})
+
+				It("should report PodIP/s as its own on interface status", func() {
+					vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
+					Eventually(vmiIP).Should(WithTransform(removeCIDR, Equal(vmiPod.Status.PodIP)), "should contain VMI Status IP as Pod status ip")
+					Eventually(vmiIPs).Should(ContainElements(vmi.Status.Interfaces[0].IP, libnet.DefaultIPv6Address), "should contain at IPs the IP and the cloud-init configured ipv6")
```

Comment:
what about ipv4?

Reply:
This is default interface test and we use guet agent, and we cannot use ValidateVMIandPodIPMatch since they don't match we have a bug at bridge binding and guest agent or the like.